app/src/pages/Lobby/Inputs/AllowCardSkipsCheckbox.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react"
+import { CurrentPlayerContext, PlayerRole } from "contexts/CurrentPlayer"
+import { CurrentGameContext } from "contexts/CurrentGame"
+import { useUpdateGameSettingsMutation } from "generated/graphql"
+import FormControlLabel from "@material-ui/core/FormControlLabel"
+import Switch from "@material-ui/core/Switch"
+
+export default function AllowCardSkipsCheckbox(props: { value: boolean }) {
+  const currentPlayer = React.useContext(CurrentPlayerContext)
+  const currentGame = React.useContext(CurrentGameContext)
+  const [updateGameSettings] = useUpdateGameSettingsMutation()
+  const [checkboxValue, setCheckboxValue] = React.useState(props.value)
+  const canConfigureSettings = currentPlayer.role === PlayerRole.Host
+
+  React.useEffect(() => {
+    setCheckboxValue(props.value)
+  }, [props.value])
+
+  return (
+    <FormControlLabel
+      control={
+        <Switch
+          checked={checkboxValue}
+          color="primary"
+          disabled={!canConfigureSettings}
+          onChange={({ target: { checked } }) => {
+            setCheckboxValue(checked)
+            updateGameSettings({
+              variables: {
+                id: currentGame.id,
+                input: { allow_card_skips: checked }
+              }
+            })
+          }}
+        />
+      }
+      label="Allow card skips"
```

Comment:
I wonder if we want to use grey[600] for the font color here, to match the rest. It sticks out bc only the headers are black

![image](https://user-images.githubusercontent.com/2534903/81506534-d9d68900-92ab-11ea-9e98-f0e4343e517f.png)

Reply:
Aside: Eventually we can draw a box around all the skip related settings... allow skips, # of skips allowed (if allow skips is true), and skips count as -1)